api/envoy/extensions/filters/network/http_connection_manager/v3alpha/http_connection_manager.proto

```diff
@@ -24,6 +24,7 @@ option java_outer_classname = "HttpConnectionManagerProto";
 option java_multiple_files = true;
 
 // [#protodoc-title: HTTP connection manager]
+
```

Comment:
You probably want to verify if this has any measurable difference in docs. The first detached comment has special treatment I think.

Reply:
+1, I think all of these need to be reverted. IIRC there can only be a single detached comment at the top of the file.